README.md

```diff
@@ -213,6 +213,93 @@ oc apply -f deploy/olm-catalog/subscription.yaml
 ```
 
 ### Creating a Windows MachineSet
+
+#### Pre-configuration steps in case of vSphere cluster:
+##### Windows VM assumptions:
+- We tested using updated version of Windows Server 1909 VM image and corresponding container images
+- We assume that the Windows image is configured with SSH so that WMCO can access it. Following are steps to comfigure SSH
+  - Configure SSH on the Windows VM using the [powershell script](https://gist.githubusercontent.com/ravisantoshgudimetla/087bf49088f70edf7f97777386d12f18/raw/0f13048697130a0265ed1050591361fdebcee3ae/powershell.yaml)
+  - Create a new file called C:\Users\Administrator\.ssh\authorized_keys and copy your public key to this file
+- We assume that the VMWare tools has been installed on the Windows VM
+- We assume that the tools.conf at C:\ProgramData\VMware\VMware Tools\tools.conf is having an entry for
```

Comment:
Documentation about configuration file location:
https://docs.vmware.com/en/VMware-Tools/11.1.0/com.vmware.vsphere.vmwaretools.doc/GUID-EA16729B-43C9-4DF9-B780-9B358E71B4AB.html

Maybe some text about if ip addresses are not available in vCenter for the guest check to make sure the service is running correct?